nservicebus/messaging/send-a-message.md

```diff
@@ -1,43 +1,54 @@
 
-Here's how a message is sent by directly using the instance of the endpoint
+## Outside the pipeline
+
+Messages are sent outside the pipeline when the send operation is not related to an incoming message. Examples may be:
+
+* Sending a command which is triggered by an HTML form submitted to an ASP.NET application.
+* Publishing an event after an user clicked a button on the GUI (see [Publish and Handle an Event](publish-subscribe/publish-handle-event.md)).
+
+A message is sent by directly using the instance of the endpoint:
 
 snippet:BasicSend
 
-And here's how it is done from the message handler
+Note: In NServiceBus Version 5 and below, `IBus` is automatically registered in the configured dependency injection container. In Version 6, `IBus` has been deprecated and replaced with `IEndpointInstance` for sends outside the pipeline. `IEndpointInstance` is not registered by default.
+
+
+## Inside the pipeline
+
+Messages are sent from within the message processing pipeline when they are sent from a message handler, a saga or some advanced extensibility points. This is code triggered by an incoming message and all send operations are linked to that incoming message. This allows transactions to rollback already sent messages when the message fails at some later stage in the pipeline.
+
+Here's how you send a simple message from a message handler:
 
 snippet:SendFromHandler
 
-In both cases you can use an interface rather than a concrete class for a message:
+It's also possible to use an interface rather than a concrete class for a message:
 
 snippet:BasicSendInterface
 
+Note: In NServiceBus Version 5 and below, the operations are available on the `IBus` which can be accessed using constructor or property injection. From version 6 `IMessageHandlerContext` is passed in directly to provide access to bus operations.
+
 
-## Immediate Dispatch
+### Immediate Dispatch
 
 While its usually best to let NServiceBus [handle exceptions for you](/nservicebus/errors), there are some scenarios where you want to send messages out even though the incoming message is rolled back. One example would be sending a reply notifying that there was an issue processing the message.
 
-In order to request immediate dispatch you can use the following syntax.
+In order to request immediate dispatch use the following syntax.
 
 snippet:RequestImmediateDispatch
 
 NOTE: By specifying immediate dispatch, outgoing messages are not [batched](/nservicebus/messaging/batched-dispatch.md) or enlisted in the current receive transaction even if the transport has support for it.
 
-
-### Suppressing the transaction scope
-
-Version 6 and below allows you to suppress the ambient transaction in order to have the outgoing message sent immediately.
+bVersion 6 and below also allows suppressing the amient transaction in order to have the outgoing message sent immediately.
```

Comment:
that sentence is from here: http://docs.particular.net/nservicebus/messaging/send-a-message#immediate-dispatch-suppressing-the-transaction-scope